src/defaultsettings.cpp

```diff
@@ -65,7 +65,7 @@ void set_default_settings()
 	settings->setDefault("max_out_chat_queue_size", "20");
 	settings->setDefault("pause_on_lost_focus", "false");
 	settings->setDefault("enable_register_confirmation", "true");
-	settings->setDefault("clickable_chat_weblinks", "false");
```

Comment:
This should be disabled on Android/ENABLE_TOUCHSCREENS still

Reply:
How would I impliment such a configuration? Thanks.

Reply:
@Froggo8311 something like this should work:
```cpp
#ifndef HAVE_TOUCHSCREENGUI
	settings->setDefault("clickable_chat_weblinks", "true");
#else
	settings->setDefault("clickable_chat_weblinks", "false");
#endif
```

Reply:
@rollerozxa Ok, thanks.

Should be right in [this commit](https://github.com/Froggo8311/minetest/commit/943239d19bc71f69a83f4324c9cf790177ad3b9f) now.